packages/ui/src/types/authenticator/stateMachine/authMachine.ts

```diff
@@ -9,7 +9,13 @@ import { AuthEvent } from './event';
 /**
  * Intefrace for `authMachine` machine interpreter
  */
-export type AuthInterpreter = Interpreter<AuthContext, any, AuthEvent>;
+export type AuthInterpreter = Interpreter<
+  AuthContext,
+  any,
+  AuthEvent,
+  any,
+  any
+>;
```

Comment:
The last two generics argument is for `xstate`'s [`TypeState`](https://xstate.js.org/docs/guides/typescript.html#typestates) and [`Typegen`](https://xstate.js.org/docs/guides/typescript.html#typegen) features. We don't use these features, but TS was getting confused when these generics weren't provided. Adding `any`s to resolve it. This should not affect actual typings we depend on.

Reply:
Can we please add a backlog task (and corresponding code comment) to address this? Even if we are not using these features currently, tightening up on `any` usage will make it much easier to transition the package over to TS strict mode when we are ready to move forward with that

Reply:
I think it'd tie with existing task here: https://app.asana.com/0/1201736086077862/1201802820587074

I'll add additional code comment.

Reply:
2f60d1a4bb0e0dd839ae79803986